lib/modules/featurePenaltyBox.js

```diff
@@ -0,0 +1,85 @@
+import * as Options from '../core/options';
+import * as Modules from '../core/modules';
+
+export const module = {};
+module.MIN_PENALTY = 1;
```

Comment:
Nah, I want these usable from the outside, like RESTips calculating `MAX_PENALTY / tips.length`